modules/brushingandlinking/src/brushingandlinkingmanager.cpp

```diff
@@ -66,17 +66,29 @@ void BrushingAndLinkingManager::setSelected(const BrushingAndLinkingInport*,
     owner_->invalidate(invalidationLevel_);
 }
 
+void BrushingAndLinkingManager::clearSelected() {
+    selected_.clear();
+    owner_->invalidate(invalidationLevel_);
+}
+
 void BrushingAndLinkingManager::setFiltered(const BrushingAndLinkingInport* src,
                                             const std::unordered_set<size_t>& indices) {
     filtered_.set(src, indices);
 }
 
+void BrushingAndLinkingManager::clearFiltered() { filtered_.clear(); }
```

Comment:
invalidation?

Reply:
that is handled internally by `filtered_`